terraform/README.md

```diff
@@ -0,0 +1,67 @@
+# Deploying with Terraform + Juju + Kubernetes
+
+The `testflinger.tf` plan in this directory can be used for deploying
+the Testflinger server into a dev, staging, or production environment.
+For development, it is recommended to use Juju with Microk8s.
+
+
+## Deploying an Environment
+
+The following assumes that you have an existing Juju + k8s environment
+with a `k8s-certification` credential imported in juju for connecting
+a new model to the k8s environment.
+
+For purposes of this example, we will be deploying a **staging**
+environment.
+
+This also requires that you import the TLS certificate for https into
```

Comment:
This would be clearer to explain where this is needed to be done (step 3), I'm not clear why this paragraph is here?

Reply:
Moved this around a bit and reworded. I also added details on how to create and import a self-signed certificate, since I think it's helpful to have it here as an optional step. Hopefully more clear now?